misc/python/materialize/mzcompose/services/materialized.py

```diff
@@ -289,6 +290,16 @@ def __init__(
             config["platform"] = platform
 
         volumes = []
+
+        if image_version is None or image_version >= "v0.140.0-dev":
```

Comment:
Does string comparison work for a Version object?

Reply:
Probably not correctly.

Reply:
it does https://python-semver.readthedocs.io/en/latest/usage/compare-versions.html